.github/workflows/run-integration-tests-basic.yml

```diff
@@ -0,0 +1,77 @@
+name: Minikube Integration Tests - basic
+on:
+   pull_request:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   push:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   workflow_dispatch:
+
+jobs:
+   run-integration-test:
+      name: Run Minikube Integration Tests
+      runs-on: ubuntu-latest
+      env:
+         KUBECONFIG: /home/runner/.kube/config
+      steps:
+         - uses: actions/checkout@v2
+
+         - name: Install dependencies
+           run: |
+              rm -rf node_modules/
+              npm install
+
+         - name: Install ncc
```

Comment:
Should be no reason to have this be a separate step. It's installed with `npm install` from the previous step.

Reply:
ok cool was wondering why we had it, it's just leftover from the previous version so I'll get rid of that